metrics/mobile-app-metrics.md

```diff
@@ -0,0 +1,108 @@
+# Abstract
+
+This document contains the proposal for gathering metrics on system statistics for running Mobile Apps and making them available to developers.
+
+## Goals
+
+### Primary Goals
+
+- Provide mobile app developers a service that can receive overall runtime metrics (processor and memory data, authentication events, runtime versions, networking events, etc).
+
+- Provide developers an interface to query and visualize the gathered data in order to make informed decisions towards deployment, deprecation of backend services and other actions that depend on the state of an application's installed base.
+
+### Secondary Goals
+
+- Provide mobile app developers with an easy to use SDK to connect easily and securely to the metrics service
```

Comment:
@david-martin @wtrocki @pb82
Should we think about security?

Reply:
At some point we will have to gather some ideas about securing the metrics endpoint. I assume it can't be linked to app auth because we want to send the initial metrics data as soon as the app launches. So it's probably something like https + API key?

I think that for now this can be left unspecified but it might be something we need to think about in the next sprint.

Reply:
`mobile-services.json` file could contain token for each individual app and  this token can be supplied for every metrics request. Token will allow to push metrics only. 

Sending email to list to gather more opinions